web-app/src/utilities/holidayStatus.js

```diff
@@ -1,8 +1,31 @@
+import React from 'react';
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+import {
+  faThumbsUp,
```

Comment:
I'm not sure everything needs an icon - that's what the colours are for. Remote working will need one because it can be one of two types, but I think the colour scheme is clear enough.

Thoughts?

Reply:
I think because remote working will need one that we should keep it consistent & always supply an icon. Also because I wanna power up my reply to you I'm gonna throw some accessibility talk in here and add that colours aren't useful if you're colour-blind.